.classpath

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="src" path="test"/>
```

Comment:
restructured the project. testing is now its own folder with each test now in the corresponding package that its testing.